src/utils/timer.cpp

```diff
@@ -33,4 +33,14 @@ uint64_t TimeDiff(const struct timeval &startTime)
     struct timeval endTime;
     gettimeofday(&endTime, NULL);
     return TimeDiff(startTime, endTime);
+}
+
+std::string DateAndTime(struct timeval &tv)
+{
+    struct tm *pTm;
+    pTm = localtime(&tv.tv_sec);
```

Comment:
it is usually better to use UTC time traces.

Reply:
As discussed offline, we want to get the same time reference as the one configured in the server.  Servers are normally configured in UTC, so everything matched.